packages/html/ds/src/progress-bar/progress-bar.schema.ts

```diff
@@ -0,0 +1,38 @@
+import * as zod from 'zod';
+
+export const progressBarSchema = zod.object({
+  currentValue: zod
```

Comment:
suggestion: you can just use `value` and for `length` for `finalValue` instead.

What do you guys think? @pregno @BogdanDanilescu

Reply:
maybe we can use `value` instead of `currentValue` and `max` instead of `finalValue` ?

Reply:
Good catch! I would go for `max` and `value` too! 

Also, I can't think of a use-case where we will have `max` as 50% or anything less than 100%. Maybe a developer can use this for size of the progress bar but again `max` would be 100% either way.